src/code/z_camera.c

```diff
@@ -4341,18 +4345,17 @@ s32 Camera_Subj4(Camera* camera) {
     Vec3f* at = &camera->at;
     u16 crawlspaceNumPoints;
     Vec3s* crawlspacePoints;
-    Vec3f sp98;
-    Vec3f sp8C;
-    f32 sp88;
-    s16 pad2;
-    f32 temp_f16;
-    PosRot sp6C;
-    VecSph sp64;
-    VecSph sp5C;
-    s16 temp_a0;
-    f32 tx;
+    Vec3f temp1;
+    Vec3f zoomAtTarget;
+    f32 temp2;
     Player* player;
-    PosRot* playerPosRot = &camera->playerPosRot;
+    f32 eyeLerp;
+    PosRot playerPosRot;
+    VecSph atEyeTargetOffset;
```

Comment:
I'm not sure `atEyeTargetOffset` is accurate?

Reply:
Oh so `atEyeTargetOffset` is the contraction of `atTargetOffset` and `eyeTargetOffset`

I was reading it like "target offset from at to eye"

Could it be just "targetOffset"? Or at least comment on the name, I think it's unclear enough (I would fall into the trap again)